vignettes/crmPack-jss-paper.Rmd

```diff
@@ -0,0 +1,842 @@
+need for either licensing specialized commercial software (thus losing
+flexibility) or programming the design and simulations from scratch for each
+project (thus losing efficiency). While the models underlying most model-based
+dose escalation procedures can easily be fit in standard software with the
+capability to fit generalized linear models, e.g., `PROC MIXED` in
+`SAS` [@SAS2003], `glm` in `Stata` [@Stata2015] or `R` [@R2016],
+there are still only few software solutions available dedicated to dose escalation studies.
+
+The commercial packages `East` [@East2016] and
+`ADDPLAN` [@Addplan2016] both offer extensions to their basic
+design software for dose escalation studies (`ESCALATE` in `East`
+and `df` in `ADDPLAN`) implementing the algorithmic 3+3 design and
+various versions of the CRM. Similarly `FACTS` [@FACTS] also
+offers different common dose escalation methods. Due to the commercial nature of
+these implementations there is, however, a limitation on how much the designs
+can be tailored towards the individual needs of the study. Similarly static
+implementations of methods for dose escalation are available in the
+`Stata` module `crm` [@StataCRM] which implements the CRM and
+the `dfcrm` package [@dfcrm2013] in `R` which additionally implements the
+time-to-event CRM (TITE-CRM) [@cheung2000]. Several `R`-packages
+with extensions are available. The `bcrm` package [@sweeting2013] implements a variety of one and two parameter models, and facilitates different
+ways to specify prior distributions, escalation and stopping rules. The `ordcrm` package
+[@ordcrm] implements ordinal proportional odds and continuation ratio models for CRMs.
+The `dfpk` package [@dfpk] uses pharmacokinetic data in the dose escalation.
+
+In this paper we introduce the `R`-package `crmPack` [@crmPack] for
+dose escalation studies, which is publicly available on CRAN. While the package's name
+pays tribute to the original CRM as the first model-based dose escalation design, the
+package's functionality differs from the above existing implementations in
+three fundamental ways. Firstly, it is written using `S4` classes and methods
+[@Chambers2008], which allows customized methodology to be added to the
+package while still being able to use the existing backbone functionalities.
+Secondly, methods for studies with a placebo group (e.g., for healthy volunteer
+studies) are readily implemented. Thirdly, dual endpoint dose escalation methods
+that incorporate both safety and efficacy and allow determination of an optimal
+biological dose are already available.
+
+# Framework
+
+For describing the framework of the package we will adapt the general notation
+for early phase trials from [@Thall2010]. Figure \@ref(fig:schematic) summarizes
+the framework in a schematic.
+
+```{r schematic, fig.align='center', out.height = "540px", out.width="700px", fig.cap='Schematic of the framework'}
+knitr::include_graphics("./schematic.pdf")
```

Comment:
This is the only imported PDF.  And yes, it is different from what's in the original Rnw.  I haven't changed the source PDF, so I assume it's rendered differently because of differences between `knitr` and `Sweave`.  I agree it's not ideal and should be changed.  But getting rid of the Rnw files should unblock some of my other (and Guana's) PRs.  So would you accept raising a new issue for me to fix it later, so we can get the other stuff moving?

Reply:
Hm can you check on google quickly, this should be a well known how to do thing?

Reply:
As a quick fix, I've taken a screenshot and saved it as a PNG.  That sorts it out for now.  But that's a pixel image, not a raster, so resolution may be an issue.